src/main/java/uk/gov/hmcts/reform/fees/client/FeeRange.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.fees.client;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import lombok.Data;
+import uk.gov.hmcts.reform.fees.client.deserialization.FeeRangeDeserializer;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Data
+@JsonDeserialize(using = FeeRangeDeserializer.class)
```

Comment:
I am not sure why we need these custom deserializer. We should try to use the default ones.

Reply:
The default deserialization will make messy nested DTOs. I'd rather have a clean API and more complicated code behind the scenes than the vice-versa.

Reply:
I don't think it should be job of a client to change the response models. Whatever service is returning it should return the same to the caller.
 
The point of making this as a separate repo is to have a reusable client which other services can also adopt. We should not change the contract/model/response dto of the endpoint in a client. Let's discuss offline if required.